docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,2465 @@
+{
```

Comment:
<p>markdown list not rendered properly here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Hm okay, for me it looks okay, how should it look like if rendered correctly?</p>